firefox-ios/Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -1072,6 +1090,35 @@ extension TelemetryWrapper {
                     value: value,
                     extras: extras)
             }
+        case(.information, .view, .urlbarImpression, _, let extras):
+            if let groups = extras?[EventExtraKey.UrlbarTelemetry.groups.rawValue]
+                as? String,
+               let interaction = extras?[EventExtraKey.UrlbarTelemetry.interaction.rawValue]
+                as? String,
```

Comment:
```suggestion
               let interaction = extras?[EventExtraKey.UrlbarTelemetry.interaction.rawValue]
                as? String,
```
Why did we put the `as? String` for all these on a separate line? It shouldn't be. It seems like there's space to not trigger the Swiftlint error.